### PR TITLE
CBP-1301 sample angular app

### PR DESCRIPTION
**Pull request by JesusRamirez2910**

### CODE REVIEW

File: src/app/app.component.ts [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  ngOnChanges() {
+    this.applyStyle()
+  }
+
+  ngOnInit() {
+    this.initFeatureFlags()
+  }
+
+  initFeatureFlags = async () => { 
+    Rox.register('demo', flags)
```
Comment (markawm) [inline src/app/app.component.ts:38]: This creates a different set of flags compared to the [react sample app](https://github.com/cloudbees-io/react-fm-example/blob/c4c3d21942da6efa5c3acd5c755eb98a869ced78/src/feature-management/FeatureFlagsProvider.tsx#L29), e.g. `demo.showMessage` vs `showMessage`.
I don't mind which, but lets standardise the flags so people can use exactly the same ones across all the sample apps.

image

**Review by markawm — COMMENTED** [review]
I don't know Angular at all tbh, but to my non-expert eye code all lgtm, @JesusRamirez2910!
I also tried running it locally which all went well. It just made me notice the namespace setting in the `register` call.

File: src/app/app.component.ts (hunk 1 above)
Comment (JesusRamirez2910) [reply src/app/app.component.ts]: updated

**Comment by JesusRamirez2910** [comment]
@markawm , updated this PR

**Review by markawm — APPROVED** [review]

**Pull request merged**